cli/command/cli.go

```diff
@@ -208,3 +208,18 @@ func getServerHost(hosts []string, tls bool) (host string, err error) {
 	host, err = opts.ParseHost(tls, host)
 	return
 }
+
+// Standard alias definitions
+var (
+	CreateAliases  = []string{"c", "cr", "cre"}
+	InspectAliases = []string{"i", "in", "ins", "insp"}
+	ListAliases    = []string{"list", "l", "li"}
+	UpdateAliases  = []string{"u", "up", "upd"}
+	RemoveAliases  = []string{"remove", "delete", "del", "d", "r"}
```

Comment:
would rm be a better choice instead of r.

Reply:
`rm` is currently the default, these aliases are in addition to it

Reply:
I agree that shortcuts should be decided upon from a product perspective (@croomes or @oicheryl ). Changing the aliases should be trivial, so I just added one that seemed sane.

Reply:
^ Cool. Awaiting Cheryl's feedback...